### ISSUE TITLE
New Feature: Add fields from iTop list view in Slack Notification

### ISSUE THREAD

**Issue by larhip** [itomig-de/itomig-webhook-integration#3]
* Add an option (AttributeEnum) to add all fields from itop list view to the notification (yes/no)
* If enabled: output should be like this:
``` 
"fields": [
			{
				"type": "mrkdwn",
				"text": "*User Request*\n<http://localhost/itop1/pages/UI.php?operation=details&class=UserRequest&id=619&|R-000619>"
			},
			{
				"type": "mrkdwn",
				"text": "*Title*\ntest 2 from UI"
			},
			{
				"type": "mrkdwn",
				"text": "*Organization*\n<http://localhost/itop1/pages/UI.php?operation=details&class=Organization&id=1&|Super-Visions>"
			},
			[...]
		] 
```

**Comment by Hipska** [comment]
Note that this can be done for Rocket.chat as well
(Implemented with 0117192)

**Comment by larhip** [comment]
Will be integrated in next release (19.3.0)

**Comment by Hipska** [comment]
When will this be released? Could it be done before iTop community meetup?

**Comment by larhip** [comment]
Hi @Hipska
It will definitely released before our community meeting. 
Right now I am just waiting for a review from a colleague. 
Best
Lars

**Comment by Hipska** [comment]
I think this can be closed since 18c9e4ca0284a9ee07b0dbe616d2801823950981.

**Comment by larhip** [comment]
Jep, you're right. I close this one.
